media/js/base/experiment-amo-init.es6.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import { getAMOExperiment } from './experiment-amo.es6.js';
```

Comment:
Note: the contents of the experiment-amo files is was converted to ES6 but is mostly the same code as before.